code/__DEFINES/obj_flags.dm

```diff
@@ -16,6 +16,7 @@
 #define INFINITE_RESKIN (1<<12) // We can reskin this item infinitely
 #define CONDUCTS_ELECTRICITY (1<<13) //! Can this object conduct electricity?
 #define NO_DECONSTRUCTION (1<<14) //! Prevent deconstruction of this object.
+#define UNWRENCH_BYPASS (1<<15) //! Can this object be unwrenched to move it around?
```

Comment:
I don't think this needs a flag, you can just override `can_be_unfasten_wrench` to return `CANT_UNFASTEN`

Reply:
Hmm actually it might be a bit more complex than I realized, let me think on this.

Reply:
can_be_unfasten_wrench gets checked inside default_unfasten_wrench after the check for the deconstruction flag, which I'm not sure we want to remove - would it be more appropriate in this case to just snowflake it for the recycler and the ORM?

Reply:
`UNWRENCH_BYPASS` serving double duty is definitely not ideal and I don't like that, I am certain there is a way we can use the procs to our advantage here somewhere but I'm just not sure how

Reply:
I'll fiddle with it and see how I can do it without the flag.

Reply:
you were dead on, works!